client/Components/Pages/ProfilePage.jsx

```diff
@@ -57,61 +58,59 @@ class ProfilePage extends Component {
 
   onSubmit(event) {
     event.preventDefault();
-
-    //************************************************************************//
-    //*THIS SECTION HANDLES THE EDITING OF PROFILE WITHOUT SELECTING AN IMAGE*//
-    //***********************************************************************//
-    this.setState({
-      imageErrorStatus: false,
-      loading: true
-    })
-    // Edit profile if there is no image selected
+    if(this.state.userDetails.fullName.trim() === '' 
```

Comment:
I believe you can assign this - `this.state.userDetails.fullName.trim()` - to a variable before the conditional. This looks to long.